tasks/elasticsearch-RedHat-version-lock.yml

```diff
@@ -2,13 +2,13 @@
 - name: RedHat - install yum-version-lock
```

Comment:
`yum-version-lock` should also be replaced here with `{{ versionlock_plugin_package }}`.

Reply:
Good call, fixed